app/views/static_pages/about.html.haml

```diff
@@ -114,4 +114,4 @@
 
   .tool
     =image_tag "tea.jpg"
-    %h4 Tea, Coffee, and Mugs
+    %h4 Tea, Coffee, and Mugs
```

Comment:
It doesn't totally matter but some interpreters of certain languages freak out when there's no last empty line at the end of a file. That's why there's a little red arrow here. I do not think this is important to fix, though.